### ISSUE TITLE
dcterms:format: how to handle content negotiation support

### ISSUE THREAD

**Issue by DavidFichtmueller** [DiSSCo/openDS#174]
### Term Name

dcterms:format

### Digital Object Name

DigitalSpecimen: Identifier

### Feedback

Definition: "Mime type of content returned by identifier in case the identifier is resolvable." 

Which mime type is returned could also depend if the hosting server supports content negotiation and serves the data in multiple formats. The definition (or potentially the usage notes) should cover how to handle this. Our recommendation would be to allow for multiple formats in form of an array (thus changing the data type of this property) and list all the ones, that the server is known to provide.

**Comment by samleeflang** [comment]
Fair point, there might be multiple Mime types in which servers can respond. I will make this an array. This will mean that we deviate from the [DWCA Extension for `Alternative Identifiers`](https://rs.gbif.org/extension/gbif/1.0/identifier.xml) in which this is a single text field. However, I don't see any issues in deviating slightly on this property.
